docs/quickstart/java.md

```diff
@@ -113,12 +112,14 @@ Keep experimenting with your model. To do so:
  @Assign
  DueDateAssigned handle(AssignDueDate command) {
      return DueDateAssignedVBuilder
-             .newBuilder()
+             .vBuilder()
              .setTaskId(command.getTaskId())
              .setDueDate(command.getDueDate())
              .build();
  }
   ```
+<p class="note">`vBuilder()` creates a Validating Builder, which checks the values against the validation options when a message is going to be build. The `newBuilder()` creates a standard Builder natively provided by Protobuf. It is safer to always use `vBuilder()`, and it does not make much sense to specify validating options, if `vBuilder()` is not used.
```

Comment:
When we replace `newBuilder()` with `vBuilder()`, then the standard Protobuf API is not visible to the user. Therefore the reference looks strange. The user needs to know Protobuf API sufficiently, and in most cases that would be not the case.

Reply:
Maybe this note should be an add-on for Validation User Guide rather than Java Quick Start Guide? Or should I better reference to a Validation Guide for more details?